common/elasticsearch/src/test/scala/uk/ac/wellcome/elasticsearch/test/fixtures/ElasticsearchFixtures.scala

```diff
@@ -48,18 +48,13 @@ trait ElasticsearchFixtures
 
   // Elasticsearch takes a while to start up so check that it actually started
   // before running tests.
-  eventually {
+  eventually(Timeout(Span(40, Seconds))) {
```

Comment:
Not exactly related to the upgrading of libs, but I think its better to just a sleep the thread a few seconds before sending the request to elasticsearch: by doing it this way we have to wait 40 seconds on a failing test just to find out it has failed.

Reply:
I don't really see the advantage. The timeout here is quite long because elasticsearch docker container is quite slow to startup especially on travis. But if you have it locally already started, this is much faster than a `Thread.sleep` because it will succeed the first time it tries. If you were to put just a `Thread.sleep` you would be forced to make it sleep 40 seconds every time you run the test locally, when in fact the long wait is only needed to cold start in CI